src/pbt/prophecy_build_tool.py

```diff
@@ -321,6 +322,15 @@ def deploy(self, fabric_ids: str = "", skip_builds: bool = False, job_ids=None):
             generate_pipeline_config_from_pipeline_component = False
 
             for component in components:
+                if "ScriptComponent" in component:
+                    script_component = component["ScriptComponent"]
```

Comment:
Can you add print log statements saying script component, to which path etc.

Reply:
done